internal/cli/up.go

```diff
@@ -55,7 +55,7 @@ func (c *upCmd) run(args []string) error {
 	go func() {
 		_ = <-sigs
 		// Sleep briefly to let tracing flush
-		time.Sleep(250 * time.Millisecond)
+		time.Sleep(500 * time.Millisecond)
```

Comment:
fwiw, the Jaeger tracer implementation has a Close method that guarantees the traces will be sent

Reply:
...oh. lol.